ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/verkle/worldview/VerkleWorldState.java

```diff
@@ -355,6 +358,16 @@ private void updateState(
             });
   }
 
+  public Set<Address> getAddressesToPersist(
+      final DiffBasedWorldStateUpdateAccumulator<?> accumulator) {
+    Set<Address> mergedAddresses =
+        new HashSet<>(accumulator.getAccountsToUpdate().keySet()); // accountsToUpdate
+    mergedAddresses.addAll(accumulator.getCodeToUpdate().keySet()); // codeToUpdate
+    mergedAddresses.addAll(accumulator.getStorageToClear()); // storageToClear
+    mergedAddresses.addAll(accumulator.getStorageToUpdate().keySet()); // storageToUpdate
```

Comment:
We need to check the performances of these merges. Depending on the number of addresses, it can have a non negligeable overhead.